includes/class-freemius.php

```diff
@@ -4827,9 +4827,9 @@ function dynamic_init( array $plugin_info ) {
                         /* translators: %s: License type (e.g. you have a professional license) */
                             $this->get_text_inline( 'You have purchased a %s license.', 'you-have-x-license' ),
                             fs_request_get( 'purchased_plan' )
-                        ) .
+                        ) . ' ' .
                         sprintf(
-                            $this->get_text_inline(" The %s's %sdownload link%s, license key, and installation instructions have been sent to %s. If you can't find the email after 5 min, please check your spam box.", 'post-purchase-email-sent-message' ),

```

Comment:
@bahiirwa please revert this and other changes with the whitespace :) I know it is not ideal, but we will lose all community translations over this. Kindly just focus on fixing the `cancel` button text. Thanks.